postgresql-dedicated-clusters-beta.mdx

```diff
@@ -0,0 +1,126 @@
+published: true
+slug: postgresql-dedicated-clusters-beta
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/images/postgresql-dedicated-clusters-beta/dedicated-clusters.jpg
+---
+
+Earlier this year we announced our [serverless Postgres platform](https://xata.io/blog/serverless-postgres-platform) and an early access program for [dedicated clusters](https://xata.io/blog/postgres-dedicated-clusters). Both of these offerings are core to Xata's evolution to becoming a Postgres database platform for faster development and a worry-free data layer. Serverless Postgres unlocks decades worth of community, tooling and support for our customers. While dedicated clusters provide more predictable costs, better performance and a more scalable solution for large production workloads. The two pair perfectly together, and we'll be diving into what's new in this blog.
+
+We're extremely excited to announce the next milestone in this evolution, our public beta for dedicated clusters 🎉
+
+We also want to whole-heartedly thank our EAP customers for not only being early adopters, but providing amazing levels of feedback throughout the journey. Keep a look out, you'll all be receiving some premium Xata swag soon 💜
+
+## What is a dedicated cluster?
+
+Today, most Xata databases live in a shared cell architecture. Our multi-tenant cells include a Postgres Aurora cluster, an Elasticsearch cluster, high availability built in and all the APIs and services required for development. Within each cell, we split the system for each workspace to include about 2,000 active databases per cell before we have to spin up a new one. Unless you were an EAP customer, your Xata database is using this shared cell architecture.
+
+Our dedicated clusters are essentially dedicated cells. So you have all the same benefits of the shared cell architecture, with less limitations caused by shared resources, more control over your infrastructure, decoupled usage based pricing and the confidence that Xata can scale with your business.
+
+![Postgres branches allocated to shared and dedicated clusters](images/branches-clusters.png)
+
+I won't get into the weeds today, but if you're interested in learning more, [this blog](https://xata.io/blog/postgres-dedicated-clusters) and [our documentation](https://xata.io/docs/dedicated-cluster) are great resources.
+
+## Get started with $1,000 in credit
+
+To kick off this [launch week](https://xata.io/launch-week-elephant-on-the-move) right and celebrate our public beta release, we will be offering **$1,000 in credit** for customers that start using dedicated cluster between now and its general availability later this year. This credit will be valid through the end of 2025, but you will only have a few months to qualify. As a customer, you do not need to apply for the credit. Any usage from a dedicated cluster will automatically be deducted and you have full control over what the deployment looks like.
```

Comment:
nit: I was thinking that this would make it more clear that you need to be on the Pro tier to take advantage of this.
 
```suggestion
To kick off this [launch week](https://xata.io/launch-week-elephant-on-the-move) right and celebrate our public beta release, we will be offering **$1,000 in credit** for Pro customers that start using dedicated cluster between now and its general availability later this year. This credit will be valid through the end of 2025, but you will only have a few months to qualify. As a customer, you do not need to apply for the credit. Any usage from a dedicated cluster will automatically be deducted and you have full control over what the deployment looks like.
```